maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -31,16 +31,8 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Do we really want blanket import?

Reply:
I think the spotless _auto_ formatter that when there are more than 10 classes imported from the same package.  This would have to be changed in maven parent's pom if we want to.

Reply:
OK, if spotless does not complain, I won't either, but that's not the usual approach for our code.